src/arguments/CorePartialDMChannel.ts

```diff
@@ -0,0 +1,21 @@
+import type { PieceContext } from '@sapphire/pieces';
+import type { PartialDMChannel } from 'discord.js';
+import { resolvePartialDMChannel } from '../lib/resolvers';
+import { Argument, ArgumentContext, ArgumentResult } from '../lib/structures/Argument';
+
+export class CoreArgument extends Argument<PartialDMChannel> {
+	public constructor(context: PieceContext) {
+		super(context, { name: 'partialDmChannel' });
```

Comment:
Following both the file's name and the method's name:
```suggestion
		super(context, { name: 'partialDMChannel' });
```